projboard/templates/projboard/searchArticle/search_article.html

```diff
@@ -0,0 +1,79 @@
+{% extends "base.html" %}
+{% load static %}
+{% block content %}
+<head>
+    <link rel="stylesheet" type="text/css" href="{% static 'css/search_article.css' %}">
+    
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />    
+    <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Anton&family=Zilla+Slab:wght@400;700&display=swap" rel="stylesheet">
+    
+</head>
+<body>
+<main>
+    <section>
+
+            <form method="POST" action="">
+                {% csrf_token %}
+                
+                <div class="search_button">
+                    <Label for="titleInput"><h4><strong>Enter Article Title:</strong></h4></Label>
+                    <div class="wrap">
+                        <span style="font-variation-settings: 'FILL' 1, 'wght' 700, 'GRAD' 0, 'opsz' 48;" class="material-symbols-outlined">search</span>
```

Comment:
Consider to move the style of this element to search_article.css.
You can set this element an id or class like you did above.

Reply:
I do not have background in FE, but i would like to know 
What are you trying to do?
Did you take a look here?
https://getbootstrap.com/docs/5.0/utilities/text/

Reply:
It's just the fonts and a bit of icons that i used right there, not realy a big deal, furthermore working with bootsrtap will not give us the design we want and the layout we want , but if you insist I can re-design it.
WDYS ?

Reply:
I choose not to die over that hill :mountain: